Shared/Samples/Run valve isolation trace/RunValveIsolationTraceView.swift

```diff
@@ -0,0 +1,182 @@
+            )
+            .onSingleTapGesture { screenPoint, mapPoint in
+                lastSingleTap = (screenPoint, mapPoint)
+            }
+            .overlay(alignment: .top) {
+                Text(model.statusText)
+                    .frame(maxWidth: .infinity, alignment: .center)
+                    .padding(10)
+                    .background(.ultraThinMaterial, ignoresSafeAreaEdges: .horizontal)
+                    .multilineTextAlignment(.center)
+            }
+            .task {
+                await model.setup()
+                await mapViewProxy.setViewpointCenter(model.startingLocationPoint, scale: 3_000)
+            }
+            .task(id: lastSingleTap?.mapPoint) {
+                guard let lastSingleTap = lastSingleTap else {
+                    return
+                }
+                if let feature = try? await mapViewProxy.identifyLayers(
+                    screenPoint: lastSingleTap.screenPoint,
+                    tolerance: 10
+                ).first?.geoElements.first as? ArcGISFeature {
+                    model.addFilterBarrier(for: feature, at: lastSingleTap.mapPoint)
+                }
+            }
+            .toolbar {
+                ToolbarItemGroup(placement: .bottomBar) {
+                    Button {
+                        isConfigurationPresented.toggle()
+                    } label: {
+                        Text("Configuration")
+                    }
+                    .disabled(model.tracingActivity == .runningTrace)
+                    Spacer()
+                    Button("Trace") {
+                        Task { await model.trace() }
+                    }
+                    .disabled(!model.traceEnabled)
+                    Spacer()
+                    Button("Reset") {
+                        model.reset()
+                        Task { await mapViewProxy.setViewpointCenter(model.startingLocationPoint, scale: 3_000) }
+                    }
+                    .disabled(!model.resetEnabled)
+                }
+            }
+            .sheet(isPresented: $isConfigurationPresented) {
+                if #available(iOS 16, *) {
+                    NavigationStack {
+                        configurationView
+                    }
+                } else {
+                    NavigationView {
+                        configurationView
+                    }
+                }
+            }
+            .overlay(alignment: .center) {
+                if model.tracingActivity != nil {
+                    VStack {
+                        Text(model.tracingActivity?.label ?? "")
+                        ProgressView()
+                            .progressViewStyle(.circular)
+                    }
+                    .padding(6)
+                    .background(.thinMaterial)
+                    .cornerRadius(10)
+                }
+            }
+            .alert(
+                "Select terminal",
```

Comment:
If I remember right, alert titles should use title case.

```suggestion
                "Select Terminal",
```